platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -140,6 +140,34 @@
     public static final int STATE_INACTIVE = 0;
     public static final int STATE_ACTIVE = 1;
 
+    // Keep track of the region state
+    private int state = STATE_INACTIVE;
+
+    private boolean deliverInactiveMessages = false;
+
+    /**
+     * Gets whether or not the `OfflineRegionObserver` will continue to deliver messages even if
+     * the region state has been set as STATE_INACTIVE.
+     */
+    public boolean isDeliverInactiveMessages() {
```

Comment:
This sounds grammatically incorrect. Is it necessary to say “isDeliver” here instead of “isDelivering” or “delivers”?